apps/omg_eth/test/fixtures.exs

```diff
@@ -24,13 +24,23 @@ defmodule OMG.Eth.Fixtures do
   deffixture eth_node do
     {:ok, exit_fn} = Eth.DevNode.start()
     on_exit(exit_fn)
+    # NOTE: The request_body will send an incrementing request "id" in each body.
+    #
+    # see: https://github.com/mana-ethereum/ethereumex/blob/649075208d2af663b9aac262b153021e960c4df8/lib/ethereumex/client/base_client.ex#L503
+    #
+    # The problem is the fixtures would send a first request out(this request). When you remove the fixtures,
+    # Ethereumex thinks we are sending the first request, missing the matching cassettes by request_body.
+    # So, we reset the counter so the cassettes can reply correctly without the fixtures:
+    :ets.insert(:rpc_requests_counter, {:rpc_counter, 0})
```

Comment:
how about splitting this one-liner into its function (can be put under `test/support`) so that the docs can be put there too. Then it will make the docs find-able for anyone looking at any of the 6 occurrences of this `insert`

Reply:
what do you think about this?